nf_core/lint.py

```diff
@@ -269,7 +269,7 @@ def check_readme(self):
             nf_config_version = self.config.get('params.nf_required_version').strip('\'"')
             try:
                 assert nf_badge_version == nf_config_version
-            except AssertionError, KeyError:
+            except (AssertionError, KeyError) as e:
```

Comment:
not necessary imo, cause the exception object is not accessed anywhere

Reply:
You're right. I guess writing `as e` is more of a reflex for me 😄